dali/operators/reader/loader/webdataset_loader.cc

```diff
@@ -34,6 +35,10 @@ inline std::string IndexFileErrMsg(const std::string& index_path, int64_t line,
 namespace detail {
 namespace wds {
 
+bool VerifyIndexVersionString(const std::string& index_string) {
```

Comment:
```suggestion
bool ValidateIndexVersionString(const std::string& index_string) {
```
It could be `Verify` if this function also checked whether this version is supported.

Answer from google:
> Rule 1: Use “validate” when referring to a process intended to establish the soundness or correctness of a construct e.g., “certificate validation”). (See: validate.) Rule 2: Use “verify” when referring to a process intended to test or prove the truth or accuracy of a fact or value (e.g., “authenticate”)

Reply:
Good point, done.